proposals/0024-opacity-micromaps.md

```diff
@@ -112,15 +115,29 @@ In HLSL under DXC, these are defined as static const uint values:
 ```hlsl
 static const uint RAYTRACING_PIPELINE_FLAG_ALLOW_OPACITY_MICROMAPS = 0x400;
 static const uint RAY_FLAG_FORCE_OMM_2_STATE = 0x400;
+static const uint RAYQUERY_FLAG_NONE = 0;
+static const uint RAYQUERY_FLAG_ALLOW_OPACITY_MICROMAPS = 0x1;
 ```
 
+Each of the above flag value definitions except `RAYQUERY_FLAG_NONE` will 
+have an availability attribute applied, restricting their use to
+shader model 6.9 and above.
+
 > Note: the fact that these flags have the same value is only a coincidence.
 
 ### Interchange Format Additions
 
-This adds no DXIL operations or metadata, it only adds a new flag value that
-may be used with existing DXIL operation parameters, returned by a DXIL
-operation.
+This adds a new DXIL operation, `AllocateRayQuery2`, to Shader Model 6.9.
+This is a new version of `AllocateRayQuery` that has a `RayQueryFlags`
+parameter corresponding to the new template argument in HLSL.
+When the `RayQueryFlags` template argument is non-zero, this new
+`AllocateRayQuery2` DXIL op is used, otherwise the current `AllocateRayQuery`
+DXIL op is used.
+
+Currently, the `AllocateRayQuery` DXIL Op has the following function signature:
+`uint [[hidden]] AllocateRayQuery(in uint flags);`
+The new DXIL Op, `AllocateRayQuery2`, will instead have this signature:
+`uint [[hidden]] AllocateRayQuery2(in uint flags, in uint ray_query_flags);`
```

Comment:
These are HLSL intrinsic signatures, and they are expressed inline instead of in a code block.  I meant that we need the DXIL signature for the `dx.op.allocateRayQuery2` operation.  Something like:

```llvm
; Function Attrs: nounwind
declare i32 @dx.op.allocateRayQuery2(i32 OpCode, i32 constRayFlags, i32 RayQueryFlags)
```